application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -104,10 +107,34 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
         if (handleIsOnCooldown(userId, event)) {
             return;
         }
+
+        TextInput body = TextInput.create(MESSAGE, "Message", TextInputStyle.PARAGRAPH)
+            .setPlaceholder("Your concerns go here")
+            .setMinLength(5)
+            .setMaxLength(1000)
+            .build();
+
+        String modMailModalComponentId =
+                generateComponentId(event.getOption(OPTION_STAY_ANONYMOUS).getAsString(),
+                        String.valueOf(event.getOption(OPTION_GUILD).getAsLong()));
+        Modal modal = Modal.create(modMailModalComponentId, "Modmail")
+            .addActionRows(ActionRow.of(body))
```

Comment:
Imo the text reads a bit odd, as if this was created by a programmer and not by someone who cared about good UX 😛

Id rename the title `Modmail` to `Send this to the moderators` and the placeholder `Your concerns go here` to `What do you want to tell them?`.

Reply:
✔️